frontend/js/src/explore/art-creator/components/IconTray.tsx

```diff
@@ -83,6 +86,29 @@ function IconTray(props: IconTrayProps) {
             </span>
           )}
         </div>
+        {browserHasClipboardAPI && (
+          <button
+            type="button"
+            className="btn btn-icon btn-info"
+            onClick={onClickCopyAlt}
+            style={{
+              marginLeft: "auto",
```

Comment:
this marginleft can be replaced with adding a `margin-right: auto` to the `.link-container` class, convenient because it already exists...